README.md

```diff
@@ -32,17 +33,17 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 - SDK version: 2.3.1
 - Build package: io.swagger.codegen.v3.generators.dotnet.CSharpClientCodegen
 
-<a name="frameworks-supported"></a>
-## Frameworks supported
+<a name="requirements"></a>
+## Requirements
 - .NET 5.0 or later
```

Comment:
Does it make sense to define some kind of support policy here?

Reply:
I've added support policy basing on Microsoft one: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core